src/Pages/jiyeon/Main/Main.js

```diff
@@ -1,13 +1,268 @@
-import React, { Component } from 'react';
+import React from "react";
+import Comment from "./Comment/Comment";
+import "./Main.scss";
 
-class Main extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
+class MainJY extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentInput: "",
+      commentData: [],
+      isBtnActive: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/commentData.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        this.setState({
+          commentData: data,
+        });
+      });
+  }
+
+  handleBtnActive = (event) => {
+    this.setState({
+      isBtnActive: this.state.commentInput.length !== 0,
+      commentInput: event.target.value,
+    });
+  };
+
+  handleCommentAdd = (event) => {
+    event.preventDefault();
+    const { commentInput, commentData } = this.state;
+    if (commentInput.length !== 0) {
+      this.setState({
+        commentData: [
+          ...commentData,
+          {
+            id: commentData.length + 1,
+            userName: "jiyeon0807",
+            content: commentInput,
+            isLiked: false,
+          },
+        ],
+        commentInput: "",
+      });
     }
-}
+  };
+
+  deleteComment = (id) => {
+    console.log("num >>> ", id);
+    // console.log("event.target >>> ", event.target);
+    const { commentData } = this.state;
+
+    const leftComments = commentData.filter((comment) => {
+      return id !== comment.id;
+    });
 
-export default Main;
\ No newline at end of file
+    this.setState({
+      commentData: leftComments,
+    });
+  };
+
+  render() {
+    const { commentInput, commentData, isBtnActive } = this.state;
+    console.log(commentData);
+
+    return (
+      <div className="Main">
```

Comment:
className을 대문자 보다는 소문자로 맞추어 주시면 좋을 것 같습니다. 
```suggestion
      <div className="main">
```

Reply:
return 에서 최상위 div는 className을 컴포먼트랑 동일하게 대문자로 하여서 Nesting 하면 좋다고 하셔서 적용해보았습니다.
컴포먼트랑 동일한 className으로 지정하면서 소문자로만 변경하면 될까요?